src/OpenTelemetry.Api.ProviderBuilderExtensions/AssemblyInfo.cs

```diff
@@ -6,8 +6,10 @@
 [assembly: InternalsVisibleTo("OpenTelemetry" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("OpenTelemetry.Tests" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("OpenTelemetry.Api.ProviderBuilderExtensions.Tests" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests" + AssemblyInfo.PublicKey)]
```

Comment:
Should this moved inside `#if !EXPOSE_EXPERIMENTAL_FEATURES`?

Reply:
Updated